vignettes/atom-plots.Rmd

```diff
@@ -1,7 +1,11 @@
 ---
 title: "Atom plots"
 author: "OSPSuiteR 2019"
-output: rmarkdown::html_vignette
+output:
```

Comment:
@IndrajeetPatil Why those changes?

Reply:
- `fig.align = 'center'` Figures aligned to center look better.
- `dpi = 300` for better resolution of plots (markdown defaults to dpi = 72, which makes for a poor figure).
- `toc = TRUE` to have a table of content, which makes it easier to navigate between different sections, which is especially useful if the vignette is long.

Reply:
awesome

Reply:
@pchelle FYI: Probably something to use for ReportingEngine as well

Reply:
Indeed